CalculatorTests/Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
These are good test cases, it's useful to cover edge cases for the list size. I think it's worth adding some cases for the other operands as well though, and trying to find edge cases for the numbers provided. For example, what happens if the user enters * and 100 0.5? What about / and 100 0?